snf-cyclades-app/synnefo/plankton/urls.py

```diff
@@ -73,6 +74,7 @@ def demux_members(request, image_id, member):
 
 urlpatterns = patterns(
     '',
+    (r'^(?:.json)?$', versions_list),
```

Comment:
You also need to support `/versions`: http://developer.openstack.org/api-ref/image/versions/index.html?expanded=list-api-versions-detail#api-versions